formatTest/unit_tests/expected_output/infix.re

```diff
@@ -948,3 +947,77 @@ let containingObject = {
       "newvalue" [@shouldBeRenderedOnString]
   }
 };
+
+let x = foo |> z;
+
+let x = foo |> f |> g;
+
+let x =
+  foo
+  |> somelongfunctionname "foo"
+  |> anotherlongfunctionname "bar" 1
+  |> somelongfunction
+  |> bazasdasdad;
+
+let code =
+  JSCodegen.Code.(
+    create
+    |> lines
+         Requires.(
+           create
+           |> import_type
+                local::"Set" source::"Set"
+           |> import_type
+                local::"Map" source::"Map"
+           |> import_type
+                local::"Immutable"
+                source::"immutable"
+           |> require
+                local::"invariant"
+                source::"invariant"
+           |> require
+                local::"Image"
+                source::"Image.react"
+           |> side_effect
+                source::"monkey_patches"
+           |> render_lines
+         )
+    |> new_line
+    |> new_line
+    |> new_line
+    |> new_line
+    |> render
+  );
+
+let code = JSCodegen.Code.(create |> render);
+
+let server = {
+  let callback _conn req body => {
+    let uri =
+      req
+      |> Request.uri
+      |> Uri.to_string
+      |> Code.string_of_uri
+      |> Server.respond
+      |> Request.uri;
+    let meth =
+      req
+      |> Request.meth
+      |> Code.string_of_method;
+    let headers =
+      req |> Request.headers |> Header.to_string;
+    body
+    |> Cohttp_lwt_body.to_string
+    >|= (
```

Comment:
Just asking, I think this PR is amazing and we should merge it before tackling on other fun problems <3